ninshiki_yolo/viewer.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2021 Ichiro ITS
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+import argparse
+import rclpy
+from rclpy.node import MsgType
+from rclpy.node import Node
+from ninshiki_interfaces.msg import YoloDetectedObjects
+
+
+class Viewer(Node):
+    def __init__(self, node_name: str, topic_name: str):
+        super().__init__(node_name)
+
+        self.image_subscription = self.create_subscription(
+            YoloDetectedObjects, topic_name, self.listener_callback, 10)
+        self.get_logger().info("subscribe image on " + self.image_subscription.topic_name)
```

Comment:
Don't you want to subscribe the image too, and draw the detection result on that image?